app/models/pydantic/creation_options.py

```diff
@@ -105,13 +139,13 @@ class VectorSourceCreationOptions(BaseModel):
         description="Include features to geostore, to make geometries searchable via geostore endpoint.",
     )
 
+    zipped: Optional[bool]
```

Comment:
I thought I got rid of it. This should now be inferred.
https://github.com/wri/gfw-data-api/blob/develop/app/tasks/vector_source_assets.py#L27

Reply:
Oh, I see. I think I re-added it because it was in a test. Will remove.